6.md

```diff
@@ -20,24 +20,30 @@ However, Prasad never took CS 2200 and is thus completely lost! Help him by
 answering some questions to design the cache.
 
 ### Q6.1 (2 points)
-How many sets are in the cache?
+How many sets are in the cache?  
+128kB / 4 ways / 64 bytes = 512 sets
 
 ### Q6.2 (3 points)
 How many bits are needed for the following (1 pt each):
 
-a) Block Offset
+a) Block Offset  
+log_2(64) = 6
 
-b) Index
+b) Index  
+log_2(512) = 9
 
-c) Tag
+c) Tag  
+64 - 9 - 6 = 49
 
 ### Q6.3 (2 points)
-How many dirty bits are needed per cache block?
+How many dirty bits are needed per cache block?  
+**8**: 1 bit for each 8 bit word
```

Comment:
byte, not bit

Reply:
For #6.5, within the discord I believe it was 20480?